docs/docs-next/docs/guides/automation.md

```diff
@@ -140,15 +82,8 @@ Asset Sensors trigger jobs when specified assets are materialized, allowing you
 - You need to trigger jobs based on asset materializations
 - You want to create dependencies between different jobs or code locations
 
-### Basic Asset Sensor Example
-
-```python
-from dagster import AssetSensor, RunRequest, asset_sensor
+For more examples of how to create asset sensors, see the [How-To Use Asset Sensors](/guides/automation/asset-sensors) guide.
 
-@asset_sensor(asset_key=["raw_data"], job=process_raw_data_job)
-def raw_data_sensor(context):
-    yield RunRequest(run_key=context.cursor)
-```
 
 ## Choosing the Right Automation Method
```

Comment:
I know a DA section was intentionally left out, but it's mentioned in the table in this section. Should it be removed for now?

Reply:
I might just add a placeholder for it below asset sensors so we don't forget with a big fat TODO ?